### ISSUE TITLE
PubCommonId: Writing with HTTP Set-Cookie

### ISSUE THREAD

**Issue by broggeri** [prebid/Prebid.js#6553]
## Question
I understand the PubCommonId implementation currently writes the cookie through the document.cookie js browser api.
Safari currently heavily limits the TTL of such cookies, in theory to 7d max, down to 24h.

I suspect it's often limited to 24h given it's often enough for the page url to have a query string to trigger the 24h limitation when the user lands from from a Google Search: https://webkit.org/blog/8828/intelligent-tracking-prevention-2-2/ 

This seems quite an existential limitation to pubcommonid - or do we think such restrictions will not be adopted by other browsers ? Or am I making a wrong assessement of the expected value of pubcommonid and 24h TTL is fine ?

In short, is there interest in adapting the spec and mechanism of pubcommonid so it would be a cookie written through HTTP Set-Cookie (which I recognize would not be as easy to deploy and as a self contained as the current prebid module)?

**Comment by patmmccann** [comment]
This already exists,  the doc is on the conversant web site

**Comment by patmmccann** [comment]
It is at https://publishersupport.conversantmedia.com/products/pubcid-pubs

The doc really needs to be moved over here

**Comment by broggeri** [comment]
Thanks, sorry for the noise then :)
